jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ImageToJsonTranslator.java

```diff
@@ -158,33 +159,31 @@ public Blob getContainerConfigurationBlob() {
       template.addHistoryEntry(historyObject);
     }
 
-    // Sets the creation time. Instant#toString() returns an ISO-8601 formatted string.
     template.setCreated(image.getCreated() == null ? null : image.getCreated().toString());
-
-    // Adds the environment variables.
     template.setContainerEnvironment(environmentMapToList(image.getEnvironment()));
-
-    // Sets the entrypoint.
     template.setContainerEntrypoint(image.getEntrypoint());
-
-    // Sets the main method arguments.
     template.setContainerCmd(image.getProgramArguments());
-
-    // Sets the exposed ports.
     template.setContainerExposedPorts(portListToMap(image.getExposedPorts()));
-
-    // Sets the volumes.
     template.setContainerVolumes(volumesListToMap(image.getVolumes()));
-
-    // Sets the labels.
     template.setContainerLabels(image.getLabels());
-
-    // Sets the working directory.
     template.setContainerWorkingDir(image.getWorkingDirectory());
-
-    // Sets the user.
     template.setContainerUser(image.getUser());
 
+    // Healthcheck is not supported by OCI
```

Comment:
I think we should have this be `== V22ManifestTemplate.class` since healthcheck is only supported by the Docker V2.2 format.